researcher_ids/RI_Claims_V1.md

```diff
@@ -1050,9 +1055,13 @@ now()+accessTokenTTL < token.exp
 Where:
 
 -   `accessTokenTTL` represents the duration for which an access token will be
-    accepted and is bounded by the next refresh token cycle and/or any larger
-    propagation delay before access is revoked, which needs to be assessed based
-    on the revocation model.
+    accepted and is bounded by the next refresh token cycle or [Claim
+    Polling](https://github.com/ga4gh/data-security/blob/master/AAI/AAIConnectProfile.md#claim-polling)
+    cycle or any larger propagation delay before access is revoked, which
+    needs to be assessed based on the revocation model.
+    
+    -   For example, if Claim Polling is performed once per hour, then
+        `accessTokenTTL` MAY be one hour.
 
 ## Token Revocation
```

Comment:
My impression is that the heading and content do not match. I understand _revocation_ means a visa or passport is cancelled before it expires. The content appears to focus on _expiration_.

Reply:
Changed to a standard refresh token example to keep things simple.